mrtt-ui/src/components/ManagementStatusAndEffectiveness/ManagementStatusAndEffectivenessForm.js

```diff
@@ -0,0 +1,375 @@
+  const validationSchema = yup.object({
+    dateOfAssessment: yup.string(),
+    stakeholderManagement: multiselectWithOtherValidationNoMinimum,
+    stakeholderInfluence: yup.string(),
+    managementStatusChanges: yup.string(),
+    currentManagementStatus: yup.string(),
+    managementLaws: yup.string(),
+    nameOfFormalManagementArea: yup.string(),
+    projectStatusChange: yup.string(),
+    currentProtectionStatus: yup.string(),
+    financeForCiteManagement: yup.string(),
+    sufficientFunds: yup.string(),
+    resourcesToEnforceRegulations: yup.string(),
+    equitableSharingOfSiteBenefits: yup.string(),
+    climageChangeAdaptation: yup.string()
+  })
+  const reactHookFormInstance = useForm({
+    defaultValues: {
+      stakeholderManagement: { selectedValues: [], otherValue: undefined }
+    },
+    resolver: yupResolver(validationSchema)
+  })
+
+  const {
+    handleSubmit: validateInputs,
+    formState: { errors },
+    reset: resetForm,
+    control,
+    watch: watchForm
+  } = reactHookFormInstance
+
+  const { siteId } = useParams()
+  const apiAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/registration_answers`
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [isSubmitError, setIsSubmitError] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
+  const managementStatusChangesWatcher = watchForm('managementStatusChanges')
+  const projectStatusChangeWatcher = watchForm('projectStatusChange')
+  const financeForCiteManagementWatcher = watchForm('financeForCiteManagement')
+
+  useInitializeQuestionMappedForm({
+    apiUrl: apiAnswersUrl,
+    questionMapping: questionMapping.managementStatusAndEffectiveness,
+    resetForm,
+    setIsLoading
+  })
+
+  const handleSubmit = (formData) => {
+    setIsSubmitting(true)
+    setIsSubmitError(false)
+
+    axios
+      .patch(apiAnswersUrl, mapDataForApi('managementStatusAndEffectiveness', formData))
+      .then(() => {
+        setIsSubmitting(false)
+        toast.success(language.success.submit)
+      })
+      .catch(() => {
+        setIsSubmitting(false)
+        setIsSubmitError(true)
+        toast.error(language.error.submit)
+      })
+  }
+
+  return isLoading ? (
+    <LoadingIndicator />
+  ) : (
+    <ContentWrapper>
+      <FormPageHeader>
+        <PageTitle>
+          {language.pages.siteQuestionsOverview.formName.managementStatusAndEffectiveness}
+        </PageTitle>
+        <PageSubtitle>{site_name}</PageSubtitle>
+      </FormPageHeader>
+      <QuestionNav
+        isSaving={isSubmitting}
+        isSaveError={isSubmitError}
+        onSave={validateInputs(handleSubmit)}
+        currentSection='management-status-and-effectiveness'
+      />
+      <Form>
+        <FormQuestionDiv>
+          <StickyFormLabel>{questions.dateOfAssessment.question}</StickyFormLabel>
+          <Controller
+            name='dateOfAssessment'
+            control={control}
+            defaultValue={new Date()}
```

Comment:
Can all the default value sbe in one place (at the top?). Also I think its better for the date to default to a null or empty string for data collection integrity. I dont think todays date is a good assumption here, and you cant clear it to be empty if the date is unknown and the user wants to come back and enter it later. Nulls are better than wrong data.

Reply:
you're right, I don't actually know what I put that default here, I think it's better as an empty string for now. Changing all the defaults at this point is a lot of work as each from uses the same methods I used in this form for defaults

Reply:
you can also set defaults in yup, which is also in some of the form sections

Reply:
Ya, not suggesting you do it everywhere. In some places it might make sense.